client/InvoiceTickets.js

```diff
@@ -10,10 +22,38 @@ Template.InvoiceTickets.helpers({
     return InvoiceTickets.byTimeRange(
       FlowRouter.getParam("filter"),
       FlowRouter.getQueryParam("sortBy"),
-      FlowRouter.getQueryParam("sortOrder")
+      FlowRouter.getQueryParam("sortOrder"),
+      Session.get("itemsLimit")
     );
   },
   formatDate: function(date){
     return moment(date).format("MM-DD-YYYY");
   },
 });
+Template.InvoiceTickets.moreResults = function() {
+    // If, once the subscription is ready, we have less rows than we
+    // asked for, we've got all the rows in the collection.
+    return !(InvoiceTickets.find().count() < Session.get("itemsLimit"));
```

Comment:
I am not trying to get the entire invoice ticket count here. It just checks if it has loaded all the invoice tickets requested before loading more tickets.
For example if my current "itemsLimit" is 40, it checks if I have loaded all 40 of them, before loading 20 new ones.